packages/amplify-category-auth/src/provider-utils/awscloudformation/service-walkthroughs/auth-questions.ts

```diff
@@ -553,19 +564,20 @@ function parseOAuthCreds(providers, metadata, envCreds) {
         }
         providerKeys[`${lowerCaseEl}AuthorizeScopes`] = provider.authorize_scopes.split(',');
       } catch (e) {
-        return null;
+        // ignore
```

Comment:
should we use AmplifyError with this error?

Reply:
```typescript
        providerKeys[`${lowerCaseEl}AuthorizeScopes`] = provider.authorize_scopes.split(',');
```
This is the only line where it is even possible to throw an error in this block, and it would be a null access error (`authorize_scopes` not found on `provider`, which is nullable from the find on line 554. The current behavior is a try catch, with the catch ignored. The last change to this line was made 4 years ago. Before introducing an AmplifyError here, I think a good first step would just be an optional chaining operator on `provider` on line 565. So line 565 would read `providerKeys[`${lowerCaseEl}AuthorizeScopes`] = provider?.authorize_scopes?.split?.(',');`

Reply:
I lied, `creds` can also be null in that block. The reasoning remains the same though. The catch behavior was to simply pass, so an optional chaining operator in each applicable place should carry the same behavior.